drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/main/java/org/drools/workbench/screens/guided/dtable/backend/server/conversion/SubHeaderBuilder.java

```diff
@@ -231,7 +237,8 @@ private void addRetract(final ActionRetractFactCol52 column) {
     public boolean addInsert(final String header,
                              final String boundName,
                              final String factType,
-                             final String factField) {
+                             final String factField,
+                             final String type) {
```

Comment:
What is the difference between `factType` and `type` please? Could we please rename the parameter to be more descriptive or add a javadoc?